src/app/public/modules/phone-field/phone-field-adapter.service.ts

```diff
@@ -0,0 +1,67 @@
+import {
+  Injectable,
+  OnDestroy,
+  Renderer2
+} from '@angular/core';
+
+import {
+  Subject
+} from 'rxjs/Subject';
+
+import {
+  SkyLibResourcesService
+} from '@skyux/i18n';
+
+@Injectable()
+export class SkyPhoneFieldAdapterService implements OnDestroy {
+
+  private ngUnsubscribe = new Subject();
+
+  constructor(
+    private renderer: Renderer2,
+    private resourcesService: SkyLibResourcesService
+  ) { }
+
+  public ngOnDestroy(): void {
+    this.ngUnsubscribe.next();
+    this.ngUnsubscribe.complete();
+  }
+
+  public addElementClass(element: HTMLElement, className: string): void {
+    this.renderer.addClass(element, className);
+  }
+
+  public setElementDisabledState(element: HTMLElement, disabled: boolean): void {
+    this.renderer.setProperty(
+      element,
+      'disabled',
+      disabled);
+  }
+
+  public setElementPlaceholder(element: HTMLElement, placeholder: string): void {
+    this.renderer.setAttribute(element, 'placeholder', placeholder);
+  }
+
+  public setElementValue(element: HTMLElement, value: string): void {
```

Comment:
We chatted about this - giving it a value makes this variable optional. So, a dev could call this method without providing a value and it would be syntactically correct, even if its not the intent. We decided to instead just wrap the whole logic with an if-check to confirm that `value` exists, instead of the ternary.